t/lib/Test/PromAlertProxy/CrashTarget.pm

```diff
@@ -0,0 +1,20 @@
+package Test::PromAlertProxy::CrashTarget;
+
+use 5.028;
+use Moo;
+use experimental qw(signatures);
+
+with 'PromAlertProxy::Target';
+
+use Types::Standard qw(Object ArrayRef);
+use Type::Utils qw(class_type);
+use Type::Params qw(compile);
+
+sub raise ($self, $alert) {
+  state $check = compile(Object, class_type('PromAlertProxy::PromAlert'));
+  $check->(@_);
```

Comment:
(lol at carefully checking your arguments before you die)

Reply:
I want my death to mean something.